Tests/test_settings_loader.py

```diff
@@ -27,39 +27,42 @@ def setUp(self):
         self.settings_file_to_test = root_dir + os.sep + "Tests/settings/settings_test.yml"
 
         self.settings_dict = {
-            'default_synapse': 'Default-synapse',
             'rest_api':
                 {'allowed_cors_origin': False,
                  'active': True,
                  'login': 'admin',
                  'password_protected': True,
                  'password': 'secret', 'port': 5000},
             'default_trigger': 'snowboy',
-            'default_player': 'mplayer',
-            'play_on_ready_notification': 'never',
             'triggers': [{'snowboy': {'pmdl_file': 'trigger/snowboy/resources/kalliope-FR-6samples.pmdl'}}],
+            'default_player': 'mplayer',
             'players': [{'mplayer': {}}, {'pyalsaaudio': {"device": "default"}}],
             'speech_to_text': [{'google': {'language': 'fr-FR'}}],
-            'on_ready_answers': ['Kalliope is ready'],
             'cache_path': '/tmp/kalliope_tts_cache',
-            'random_wake_up_answers': ['Oui monsieur?'],
-            'on_ready_sounds': ['sounds/ding.wav', 'sounds/dong.wav'],
             'resource_directory': {
                 'stt': '/tmp/kalliope/tests/kalliope_resources_dir/stt',
                 'tts': '/tmp/kalliope/tests/kalliope_resources_dir/tts',
                 'neuron': '/tmp/kalliope/tests/kalliope_resources_dir/neurons',
                 'trigger': '/tmp/kalliope/tests/kalliope_resources_dir/trigger'},
             'default_text_to_speech': 'pico2wave',
             'default_speech_to_text': 'google',
-            'random_wake_up_sounds': ['sounds/ding.wav', 'sounds/dong.wav'],
             'text_to_speech': [
                 {'pico2wave': {'cache': True, 'language': 'fr-FR'}},
                 {'voxygen': {'voice': 'Agnes', 'cache': True}}
             ],
             'var_files': ["../Tests/settings/variables.yml"],
             'start_options': {
                 'muted': True
-            }
+            },
+            'hooks': {'on_waiting_for_trigger': 'test',
```

Comment:
Missing "on_start_speaking" and "on_stop_speaking" hooks ?

Reply:
the user can let them empty. It's not a big deal to not add them into the settings test.